sys/Makefile.dep

```diff
@@ -315,7 +315,7 @@ ifneq (,$(filter shell_commands,$(USEMODULE)))
   endif
 
   ifneq (,$(filter gnrc_icmpv6_echo,$(USEMODULE)))
-    USEMODULE += netutils
+    USEMODULE += netutils ztimer_usec
```

Comment:
I don't like this. The code of `gnrc_icmpv6_echo` does not require `ztimer`, but the shell command which implements `ping` does (and that is how the dependency used to be expressed). Pulling in `ztimer_usec` here as a dependency can have unintentional repercussions for testing, so I rather prefer to keep the dependency as is: Pull in `sc_gnrc_icmpv6_echo.c`, when a timer module is present.

Reply:
Also take note that `gnrc_icmpv6_echo` is also used for the "ICMPv6 ping server" side, where no notion of time is needed => a possibly unneeded timer module is pulled in.

Reply:
Note that this is all inside

```Makefile
ifneq (,$(filter shell_commands,$(USEMODULE)))
  [...]
endif
```

if the user wants shell commands, the user should get shell commands. (And adding a fine-grained selection of shell commands would IMO be a welcome addition, but is not the goal of this PR.)

Reply:
Ah, sorry I missed that context!

Reply:
Then all good.